pkg/plugins/policies/meshratelimit/api/v1alpha1/deprecated.go

```diff
@@ -9,6 +11,35 @@ func (t *MeshRateLimitResource) Deprecations() []string {
 	var deprecations []string
 	if len(pointer.Deref(t.Spec.From)) > 0 {
 		deprecations = append(deprecations, "'from' field is deprecated, use 'rules' instead")
+		for i, rule := range pointer.Deref(t.Spec.From) {
+			if rule.Default.Local != nil && isStatusInvalid(pointer.Deref(rule.Default.Local)) {
```

Comment:
you don't need pointer.Deref if you just checked that `rule.Default.Local` is not nil